.github/workflows/reproducible.yml

```
@@ -30,7 +30,7 @@ jobs:
     #               varies based on outside environment.
 
     # Skip this for PRs
-    if: github.event_name != 'pull_request'
+    #if: github.event_name != 'pull_request'
```

```suggestion
    if: github.event_name != 'pull_request'
```

So we remember to change it back.